engine-api/src/main/java/pl/edu/icm/unity/engine/api/AttributesManagement.java

```diff
@@ -99,4 +101,9 @@
 	Collection<AttributeExt> getAllAttributes(EntityParam entity, boolean effective, 
 			String groupPath, String attributeTypeId, boolean allowDegrade) throws EngineException;
 
+	/**
+	 * Returns attributes of an entity, including hidden ones.
```

Comment:
The key thing here is that attributes from multiple groups are returned, and groups are specified by patterns.

Reply:
done